playbooks/roles/airship-setup-deployer/tasks/main.yml

```diff
@@ -17,6 +17,17 @@
   tags:
     - install
 
+- name: Install client packages
+  package:
+    name: "{{ suse_osh_client_packages }}"
```

Comment:
Does that work given that `{{ suse_osh_client_packages }}` is a list?

Reply:
The whole block is basically copy an paste of the `suse_airship_deploy_packages` installation, so I'd say yes.

Reply:
wondering if it is because only one item in the list?

Reply:
I actually plan this list will include helm package once we have it packaged

Reply:
package action plugin flattens out lists, so it's good if you have one or many packages.